ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/NoSemicolonsRule.kt

```diff
@@ -73,6 +78,21 @@ class NoSemicolonsRule : Rule("no-semi") {
         return nextLeaf == null /* eof */
     }
 
+    private fun doesNotRequirePostSemi(prevLeaf: ASTNode?): Boolean {
+        if (prevLeaf?.elementType == OBJECT_KEYWORD) {
+            // https://github.com/shyiko/ktlint/issues/281
+            return false
+        }
+        val parent = prevLeaf?.treeParent?.psi
```

Comment:
could you also drop a comment, like the one above, which is referencing the issue (#955)?